docs/libs/ml/svm.md

```diff
@@ -186,5 +199,5 @@ svm.fit(trainingDS)
 val testingDS: DataSet[Vector] = env.readVectorFile(pathToTestingFile)
 
 // Calculate the predictions for the testing data set
-val predictionDS: DataSet[LabeledVector] = svm.predict(testingDS)
+val predictionDS: DataSet[(FlinkVector, Double)] = svm.predict(testingDS)
```

Comment:
Maybe we shouldn't use renaming of imports here. This might be confusing for people not so familiar with Scala.